component/background/RoomService.tsx

```diff
@@ -11,11 +12,16 @@ const config = {
 };
 firebase.initializeApp(config);
 
-class RoomService {
+const allRooms = firebase.database().ref("rooms");
+const allUsers = firebase.database().ref("users");
+
+const MESSAGE = "message";
+const USER = "user";
```

Comment:
This does not answer the question. Why do they need to be global rather than private variables within the class?